src/components/List_Products.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+import { getItems, saveItem } from '../lib/api';
+
+function List_Products() {
+  const [Items, setItems] = useState(null);
+  const [ItemName, setItemName] = useState(null);
+
+  /* Get Items */
+  useEffect(() => {
+    getItemsData();
+  }, []);
```

Comment:
I'm not sure totally. However, I believe that **AC 2 is pending.** When I update the Firebase database from Firebase's console, the application **doesn't change**.  I suggest reviewing [this resource](https://firebase.google.com/docs/firestore/query-data/listen#listen_to_multiple_documents_in_a_collection) (it's about listening collection changes `onSnapshot`) 👀 

❓ Am I right? or Do you have another opinion?

Reply:
I was wondering about how to better coordinate effects triggered by db changes and effects triggered by state changes in React.
I included onSnapshot  in a new commit and now db changes are reflected in the app, but there's no unsubscription from the listener and I'm not really sure if there's also a best practice from what I implemented.

Reply:
Added the unsubscribe() in a new commit.